tools/behave_rule.bzl

```diff
@@ -61,18 +61,49 @@ def _rule_implementation(ctx):
            fi
            DIRECTORY=$(ls ./grakn_distribution)
            echo Successfully unarchived Grakn distribution.
+
+           RND=20001
+           while [ $RND -gt 20000 ]  # Guarantee fair distribution of random ports
+           do
+             RND=$RANDOM
+           done
+           PORT=$((40000 + $RND))
+
            echo Starting Grakn Server
            mkdir ./grakn_distribution/"$DIRECTORY"/grakn_test
-           ./grakn_distribution/"$DIRECTORY"/grakn server --data grakn_test &
-           sleep 9
+           ./grakn_distribution/"$DIRECTORY"/grakn server --port $PORT --data grakn_test &
+
+           POLL_INTERVAL_SECS=0.5
+           MAX_RETRIES=60
+           RETRY_NUM=0
+           while [ $RETRY_NUM -lt $MAX_RETRIES ]
+           do
+             ((RETRY_NUM++))
+             if [ $(($RETRY_NUM % 4)) -eq 0 ]
+             then
+               echo Waiting for Grakn server to start \($(($RETRY_NUM / 2))s\)...
```

Comment:
I wanted to write `$RETRY_NUM * $POLL_INTERVAL_SECS`, but Bash doesn't support floating-point arithmetic (although it _is_ able to divide one integer by another)